tools/api-markdown-documenter/src/documentation-domain/AlertNode.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { DocumentationNode, ParentNodeBase } from "./DocumentationNode";
+import { DocumentationNodeType } from "./DocumentationNodeType";
+import { createNodesFromPlainText } from "./Utilities";
+
+// TODO: This type really doesn't belong here. It isn't a core Documentation concept.
+// The FluidFramework website's build should define this as a custom transformation policy for `beta` and `deprecation` notices instead.
+
+/**
+ * Kind of alert.
+ */
+export enum AlertKind {
+	/**
+	 * A suggestion or useful tip for the reader.
+	 */
+	Tip = "Tip",
+
+	/**
+	 * A general note for the user.
+	 */
+	Note = "Note",
+
+	/**
+	 * An important note for the user.
+	 */
+	Important = "Important",
+
+	/**
+	 * A precautionary warning for the user.
+	 */
+	Warning = "Warning",
+
+	/**
+	 * A serious precautionary warning for the user.
+	 */
+	Danger = "Danger",
+}
+
+/**
+ * An highlighted notice about nearby content for the user.
+ *
+ * @remarks See {@link AlertKind} for a list of supported alert kinds.
+ *
+ * @example Markdown
+ *
+ * ```md
+ * > [TIP]: Unit tests are super useful!
+ * >
+ * > More details about unit tests...
+ * ```
+ *
+ * @example HTML
+ *
+ * ```html
+ * <blockquote>
+ * 	<b>[TIP]: Unit tests are super useful!</b>
+ * 	<br>
+ * 	<br>
+ * 	More details about unit tests...
+ * </blockquote>
```

Comment:
This was the default policy I created to line up with the default Markdown policy. Since Alerts are not actually well defined in the system (vanilla Markdown doesn't have a concept of them). Our website build provides a custom render override for AlertNodes that generates Hugo-compatible alerts.

This preserves the rendering behavior we have previously, so I'm inclined to leave it alone for now, and address by removing it altogether later. But let me know if you have strong feeling in opposition to this.

Reply:
As long as blockquotes for this purpose don't make it out to ff.com, I'm happy.

Reply:
Relevant discussion and links from the past (just for posterity): https://github.com/microsoft/FluidFramework/pull/13053#discussion_r1029944233